JSDemos/Demos/Diagram/WebAPIService/description.md

```diff
@@ -1 +1 @@
-This demo shows the **Diagram** widget loading data from a Web API service. 
+This example demonstrates the **Diagram** widget communicating with a Web API service. All data editing and data processing operations are delegated to the server. The <a href="https://github.com/DevExpress/DevExtreme.AspNet.Data#devextreme-aspnet-data" target="_blank">DevExtreme.AspNet.Data</a> extension’s <a href="https://github.com/DevExpress/DevExtreme.AspNet.Data/blob/master/docs/client-side-with-jquery.md#api-reference" target="_blank">createStore</a> method configures access to the server from the client. The server implementation is available under the _DiagramEmployeesController.cs_ tab in the ASP.NET MVC version of this demo.
```

Comment:
"the **Diagram** widget communicating with a Web API service" Explain. How can you demonstrate 'communicating'?

Reply:
"All data editing and data processing operations are delegated to the server" -> All data editing and processing operations are delegated to the server